code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -520,7 +520,7 @@
 		),
 		"Grenades" = list(
 			/obj/item/weapon/gun/grenade_launcher/single_shot = -1,
-			/obj/item/weapon/gun/grenade_launcher/multinade_launcher = -1,
+			/obj/item/weapon/gun/grenade_launcher/multinade_launcher/unloaded = -1,
```

Comment:
It's kind of funny to include valhalla in this change but it's not like it particularly matters.

Reply:
If I didn't, it would mess people up trying to save loadouts in valhalla